ModelA Accuracy: 0.000
ModelB Top-5 Accuracy: 0.000
Traceback (most recent call last):
  File "/home/vagrant/starlight/scripts/train_gru_model_v3.py", line 144, in <module>
    main()
  File "/home/vagrant/starlight/scripts/train_gru_model_v3.py", line 140, in main
    result = combined_inference(modelA, modelB, cmd_input, max_steps=5)
  File "/home/vagrant/starlight/scripts/train_gru_model_v3.py", line 105, in combined_inference
    next_file_pred = modelB(cmd_input, file_hist_tensor).argmax(dim=1)
  File "/home/vagrant/starlight/scripts/.venv/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1736, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/vagrant/starlight/scripts/.venv/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1747, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/vagrant/starlight/scripts/train_gru_model_v3.py", line 32, in forward
    combined = torch.cat([cmd_emb.repeat(1, file_emb.size(1), 1), file_emb], dim=2)
RuntimeError: Number of dimensions of repeat dims can not be smaller than number of dimensions of tensor
